etc/scripts/docenizers/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "lib": ["es2021"],
```

Comment:
This is needed for `String.prototype.replaceAll`. Would be worth looking into for the backend tsconfig as well (when I tried it caused `performance` to not be recognized though)